PelotonEppSdk/Models/CreditCardTransactionRequest.cs

```diff
@@ -0,0 +1,205 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using PelotonEppSdk.Classes;
+using PelotonEppSdk.Enums;
+
+namespace PelotonEppSdk.Models
+{
+    public class CreditCardTransactionRequest : RequestBase
```

Comment:
Should the following be required?
`CardOwner`, `CardNumber`, `ExpiryMonth`, `ExpiryYear`, `CardSecurityCode`, `Amount` be required?

Reply:
It ought to match the API specifications [documented here](https://api.peloton-technologies.com/Help/Api/POST-v1-CreditCardTransactions). It looks like those fields are not required in that documentation. In practice, I'm not sure that that is true, but that's beside the point.

Reply:
In that case, looks good to me.